firstsection.css

```diff
@@ -0,0 +1,542 @@
+.first {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+.content {
+    overflow-x: hidden;
+    overflow-y: visible;
+}
+.content__banner {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    position: relative;
+    margin-bottom: 43px;
+    text-transform: uppercase;
+    color: white;
+}
+.banner {
+    width: 100%;
+}
+.banner__img {
+    object-fit: cover;
+    object-position: center;
+    width: 100%;
+}
+.banner__title {
+    font: bold 30px/29px 'Gotham Pro';
+    position: absolute;
+    top: 76px;
+    left: 54px;
+    width: 209px;
+    margin: 0;
+}
+.banner__shop {
+    font: bold 12px 'Gotham Pro';
+    position: absolute;
+    top: 157px;
+    left: 54px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/final-Axion/pullRequest?prid=5455661)